modes/ebib/evil-collection-ebib.el

```diff
@@ -47,6 +47,7 @@
     "k" 'ebib-prev-entry
     "n" 'ebib-search-next
     "p" 'ebib-yank-entry
+    "gj" 'ebib-jump-to-entry
```

Comment:
IMHO, `J` is better than `gj`.

Reply:
`J` has been used by ebib for jump to other database.

Reply:
There is no `J` binding defined in `ebib-index-mode-map` and `ebib-entry-mode-map`.

Reply:
[https://github.com/joostkremers/ebib/blob/37b9c4ec7a57ffdda53f3345cc99cf6819e94863/ebib.el#L966](source)

Reply:
OK